terraform/modules/tf_stream_alert_athena/variables.tf

```diff
@@ -37,3 +37,8 @@ variable "athena_data_buckets" {
 variable "prefix" {
   type = "string"
 }
+
+variable "refresh_interval" {
+  type    = "string"
+  default = "rate(10 minutes)"
```

Comment:
good catch, I actually just made this change and forgot to add it to the PR body